daemon/graphdriver/overlay/overlay.go

```diff
@@ -12,6 +12,8 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/docker/docker/errdefs"
+
```

Comment:
nit: should be grouped with the other imports below

Reply:
Fixed